lib/wifi_user/use_case/sponsor_users.rb

```diff
@@ -62,14 +63,16 @@ def sponsor_email(sponsor, sponsor_address, sponsee_address)
       personalisation: login_details.merge(sponsor: sponsor),
       email_reply_to_id: do_not_reply_email_address_id
     )
+    # FIXME: we currently don't support checking if the email was invalid, so always say it is
```

Comment:
Is this for a later PR?

Reply:
Not sure how I feel about FIXME's ending up on `master`. Feel like we've not had that discussion as a team ever!

Reply:
there definitely needs to be some comment about why this returns true.

Starting to see that we should update the signature of all theses affected functions to make it clearer that they all now cascade upwards to report failures.

Reply:
Yeah I don't like the FIXME's Im down to get rid